### PR TITLE
React week1

### PR DESCRIPTION
**Pull request by hwaleedali**

### CODE REVIEW

File: react/react1/week1/my-page/src/App.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import logo from "./logo.svg";
+import "./App.css";
+import { TodoList } from "./TodoList";
```
Comment (TITHI1244) [inline react/react1/week1/my-page/src/App.js]: Though it does the exact same thing, but I hope you know that if you want to import it as Default you could do it by `import TodoList from "./TodoList";` and mention export default on that component. As TodoList has only that.

File: react/react1/week1/my-page/src/App.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  { description: "Get out of bed", deadline: "Wed Sep 13 2017" },
+  { description: "Brush teeth", deadline: "Thu Sep 14 2017" },
+  { description: "Eat breakfast", deadline: "Fri Sep 15 2017" },
+];
+
+function App() {
+  return (
+    <div className="App">
+      <h3>Todo List</h3>
+      <TodoList TodoData={TodoData}></TodoList>;
```
Comment (TITHI1244) [inline react/react1/week1/my-page/src/App.js]: Usually we don't name the props as Pascal case, rather camel case. Pascal case is for components.

File: react/react1/week1/my-page/src/TodoItems.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import React from "react";
+function TodoItem(props) {
```
Comment (TITHI1244) [inline react/react1/week1/my-page/src/TodoItems.js]: You might check Object destructuring here, and use it like `{ description, deadline }` and thus you might use it directly. A cool feature.

**Review by TITHI1244 — APPROVED** [review]
Nicely done. Added a few comments to add it on your future checklist.

File: react/react1/week1/my-page/src/TodoItems.js (hunk 3 above)
Comment (hwaleedali) [reply react/react1/week1/my-page/src/TodoItems.js]: done everything

**Pull request merged**